app/components/Settings/OptionsDropdown/sql-options.jsx

```diff
@@ -33,7 +33,7 @@ export default class SQLOptions extends Component {
             return (
                 <div>
                     <div>{'Hm.. there was an error loading up your tables'}</div>
-                    <div style={{color: 'red'}}>{JSON.stringify(tablesRequest)}</div>
+                    <div style={{color: '#EF553B'}}>{JSON.stringify(tablesRequest)}</div>
```

Comment:
Just a thought.  Maybe define '#EF553B' as a constant. Not a requirements

Reply:
Good catch! Updated ✔️